Add large Swiftsong transport jobs (#9575)
endless-sky/endless-sky: data/hai/hai jobs.txt

		payment 15000 150
		"reputation: Hai" += 5
		dialog "The Hai depart your ship chittering in excitement. One of them hands you your payment of <payment> before running to catch up with their friends."

mission "Swiftsong Business Convention [1]"

> 	name "Transport to business convention at <planet>"

Reviewer: ```suggestion
	name "<planet> business convention"
```

	job
	repeat
	description "A group of <bunks> Hai who need to get to a business convention will pay you <payment> in exchange for transporting them and their <tons> of supplies to <planet>."
	passengers 4 2 .1
	cargo "office supplies" 1 2
	to offer
		random < 10
	source
		government "Hai"

> 		not planet "Swiftsong"

Reviewer 2: Would it be better to set a minimum distance of 2? Otherwise, you might get jobs from Ya Hai. You might also get them from Icelake, which is in the same system? I'm not sure if that's intended.

Author: Good point but I don't really see this as a problem tbh, there's gotta be people from Ya Hai or Icelake who want to go to events on Swiftsong.

	destination "Swiftsong"
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on complete

> 		payment 10000 500

Reviewer: ```suggestion
		payment 10000 150
```

		"reputation: Hai" += 3

> 		dialog `During the trip, your passengers appeared to be practicing a presentation for the conference. You sense a tinge of nervousness as they give you <payment> and leave your ship.`

Reviewer: ```suggestion
		dialog `During the trip, your passengers appeared to be practicing a presentation for the convention. You sense a tinge of nervousness as they give you <payment> and leave your ship.`
```

mission "Swiftsong Business Convention [2]"

> 	name "Transport to business convention at <planet>"

Reviewer: Do the same for the rest.
```suggestion
	name "<planet> business convention at"
```

	job
	repeat
	description "A group of <bunks> Hai who need to get to a business convention will pay you <payment> in exchange for transporting them and their <tons> of supplies to <planet>."
	passengers 4 4 .1
	cargo "office supplies" 1 3
	to offer
		random < 5
	source
		government "Hai"
		not planet "Swiftsong"
	destination "Swiftsong"
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on complete

> 		payment 10000 500

Reviewer: Do the same for the rest.
```suggestion
		payment 10000 150
```

		"reputation: Hai" += 3

> 		dialog `During the trip, your passengers appeared to be practicing a presentation for the conference. You sense a tinge of nervousness as they give you <payment> and leave your ship.`

Reviewer: ```suggestion
		dialog `During the trip, your passengers appeared to be practicing a presentation for the convention. You sense a tinge of nervousness as they give you <payment> and leave your ship.`
```

mission "Swiftsong Film Festival [1]"

> 	name "Transport to film festival at <planet>"
> 	job
> 	repeat

Reviewer: For all of these, have the one-word tags first. We also tend to keep the name and description next to one another.
```suggestion
	job
	repeat
	name "Transport to film festival at <planet>"
```

	name "Transport to science fiction convention at <planet>"
	job
	repeat
	description "A group of <bunks> Hai will pay you <payment> in exchange for transporting them to <planet>, where a science fiction convention is taking place."
	passengers 4 2 .1
	to offer
		random < 10
	source
		government "Hai"
		not planet "Swiftsong"
	destination "Swiftsong"
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment 10000 500
		"reputation: Hai" += 3
		dialog `The Hai chitter excitedly as they give you <payment> and step off your ship. As they leave, you can identify some of their memorabilia as being human in origin.`

mission "Swiftsong Science Fiction Convention [2]"
	name "Transport to science fiction convention at <planet>"
	job
	repeat
	description "A group of <bunks> Hai will pay you <payment> in exchange for transporting them to <planet>, where a large science fiction convention is taking place."
	passengers 4 4 .1
	to offer
		random < 5
	source
		government "Hai"
		not planet "Swiftsong"
	destination "Swiftsong"
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment 10000 500
		"reputation: Hai" += 3

> 		dialog `The Hai chitter excitedly as they give you <payment> and step off your ship. As they leave, you can identify some of their memorabilia as being human in origin.`

Reviewer: ```suggestion
		dialog `The Hai chitter excitedly as they give you <payment> and step off your ship. As they leave, one of them gives you a hand gesture that you recognize from an old human sci-fi program.`
```

		payment 15000 150
		"reputation: Hai" += 5
		dialog "The Hai depart your ship chittering in excitement. One of them hands you your payment of <payment> before running to catch up with their friends."

mission "Swiftsong Business Convention [1]"
	name "Transport to business convention at <planet>"
	job
	repeat
	description "A group of <bunks> Hai who need to get to a business convention will pay you <payment> for you to transport them and their <tons> of supplies to <planet>."
	passengers 4 2 .1
	cargo "office supplies" 1 2
	to offer
		random < 60
	source
		government "Hai"
	destination "Swiftsong"
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on complete
		payment 10000 500
		"reputation: Hai" += 3
		"reputation: Hai (Wormhole Access)" += 3
		"reputation: Hai Merchant" += 3
		"reputation: Hai Merchant (Human)" += 3
		"reputation: Hai Merchant (Sympathizers)" += 3

> 		dialog phrase "generic passenger on visit"

Reviewer 3: Should these dialog phrases be here?

Author: Whoops, should be fixed now.
